ydb/tests/stress/viewer/workload/__init__.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
+import time
+import threading
+import queue
+import traceback
+import requests
+from urllib.parse import urljoin
+from concurrent.futures import ThreadPoolExecutor
+
+class Workload:
+    def __init__(self, mon_endpoint, database, duration):
+        self.mon_endpoint = mon_endpoint
+        self.database = database
+        self.round_size = 1000
+        self.duration = duration
+        self.delayed_events = queue.Queue()
+        self.pool_semaphore = threading.BoundedSemaphore(value=100)
+        self.worker_exception = []
+        self.pool = None
+
+    def wrapper(self, f, *args, **kwargs):
+        try:
+            if len(self.worker_exception) == 0:
+                result = f(*args, **kwargs)
+                if result.status_code != 200:
+                    raise AssertionError(f"Bad response code {result.status_code}, text: {result.text}")
+        except Exception:
+            self.worker_exception.append(traceback.format_exc())
```

Comment:
Вот тут мне кажется можно обернуть в блокировку, на GIL я бы не рассчитывал
self.lock = threading.Lock()


```suggestion
            with self.lock:
                self.worker_exception.append(traceback.format_exc())
```